tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -279,6 +282,11 @@ def get_mimetext_message(self, sqs_message, resources, to_addrs):
         return message
 
     def send_c7n_email(self, sqs_message, email_to_addrs, mimetext_msg):
+
+        # append cc addresses to outgoing list
+        if sqs_message['action'].get('cc'):
+            email_to_addrs += sqs_message['action']['cc']
```

Comment:
same thing as the above comment, need to resolve email addrs before adding them to email_to_addrs